asabiris/handlers/webhandler.py

```diff
@@ -105,20 +107,32 @@ async def send_email(self, request, *, json_data):
 				body_params=json_data["body"].get("params", {}),  # Optional
 				attachments=json_data.get("attachments", []),  # Optional
 			)
-		except KeyError as e:
-			raise aiohttp.web.HTTPNotFound(text="{}".format(e))
-
-		except jinja2.exceptions.UndefinedError as e:
-			raise aiohttp.web.HTTPBadRequest(text="Jinja2 error: {}".format(e))
-
-		except SMTPDeliverError:
-			raise aiohttp.web.HTTPServiceUnavailable(text="SMTP error")
-
-		except PathError as e:
-			raise aiohttp.web.HTTPNotFound(text="{}".format(e))
+		except ASABIrisError as e:
+			response = {
+				"result": "ERROR",
+				"error": e.Errori18nKey,
+				"error_dict": e.ErrorDict,
+				"tech_err": e.TechMessage
+			}
+			return aiohttp.web.json_response(response, status=400)
```

Comment:
Maybe add some entry to logs as well?

Reply:
@language-shprt Do we really need it?
It can spike unwanted alerts?